repose-aggregator/components/filters/valkyrie-authorization-filter/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -321,27 +325,59 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
       } else {
         s"/account/$transformedTenant/permissions/contacts/$callType/by_contact/$contactId/effective"
       }
-      Try(akkaServiceClient.get(cacheKey(callType, transformedTenant, contactId),
-        valkyrieServer.getUri + uri,
-        Map("X-Auth-User" -> valkyrieServer.getUsername, "X-Auth-Token" -> valkyrieServer.getPassword) ++ requestTracingHeader)
-      )
+      (Option(valkyrieServer.getUsername), Option(valkyrieServer.getPassword)) match {
+        case (Some(username), Some(password)) =>
+          Try(akkaServiceClient.get(cacheKey(callType, transformedTenant, contactId),
+            valkyrieServer.getUri + uri,
+            Map("X-Auth-User" -> username, "X-Auth-Token" -> password) ++ requestTracingHeader)
+          )
+        case _ =>
+          Try(akkaServiceClient.get(cacheKey(callType, transformedTenant, contactId),
+            valkyrieServer.getUri + uri,
+            Map("X-Auth-Token" -> authToken.getOrElse(null)) ++ requestTracingHeader)
```

Comment:
Yeah, it works, but not looking at it has given me a better way to handle it. Update inbound.